glyphs-reader/src/font.rs

```diff
@@ -67,15 +67,33 @@ pub struct Glyph {
     pub other_stuff: BTreeMap<String, Plist>,
 }
 
+/// Represents a path OR a component
+///
+/// <https://github.com/schriftgestalt/GlyphsSDK/blob/Glyphs3/GlyphsFileFormat/GlyphsFileFormatv3.md#differences-between-version-2>
+#[derive(Clone, Debug, FromPlist, PartialEq, Eq, Hash)]
+pub struct Shape {
+    // When I'm a path
+    pub closed: Option<bool>,
+    pub nodes: Option<Vec<Node>>,
+
+    // When I'm a component I have a ref
+    // but it seems to end up other_stuff because r#ref doesn't work
+    pub pos: Option<Vec<OrderedFloat<f64>>>,
```

Comment:
there's also scale and angle..

Reply:
There's *tons* of attributes we don't handle yet, will add a comment.